servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRoutes.java

```diff
@@ -209,8 +228,8 @@ private GrpcExecutionStrategy executionStrategy(final String path, final Method
             final Class<Req> requestClass, final Class<Resp> responseClass,
             final GrpcSerializationProvider serializationProvider) {
         final Method method = retrieveMethod(serviceClass, methodName, GrpcServiceContext.class, requestClass);
-        routeBuilder.addRoute(path, executionStrategy(path, method, serviceClass), route, requestClass, responseClass,
-                serializationProvider);
+        routeBuilder.addRoute(path, executionStrategy(path, method, serviceClass), supportedEncodings, route,
```

Comment:
Lets hold on, on this excitement :) 
The code is ready to support it, but in order to do so, we have to expose options for each route on the factory. I don't want to commit on an API that we still don't know who and how it will be used. Unless you have hard objections, I would like to keep it simpler for now. WDYT?

Reply:
ok ya makes sense.